stake-pool/cli/src/main.rs

```diff
@@ -696,7 +841,7 @@ fn command_list(config: &Config, stake_pool_address: &Pubkey) -> CommandResult {
         println!("Validator List: {}", stake_pool.validator_list);
         println!("Manager: {}", stake_pool.manager);
         println!("Staker: {}", stake_pool.staker);
-        println!("Depositor: {}", stake_pool.deposit_authority);
+        println!("Depositor: {}", stake_pool.stake_deposit_authority);
```

Comment:
Can we print the sol deposit authority if it exists? And also the deposit, withdrawal, and referral fees?

Reply:
yep, didn't realise this functionality existed!